### ISSUE TITLE
Accessibility Checklist

### ISSUE THREAD

**Issue by anssiko** [w3c/compute-pressure#183]
This issue is a record of the [Devices and Sensors Working Group](https://www.w3.org/das/)'s response to the [Accessibility Checklist](https://w3c.github.io/apa/fast/checklist.html) for the [Compute Pressure API](https://www.w3.org/TR/compute-pressure/). Completed Checklist is required for the submission of the Accessibility review, one of the wide reviews tracked in #177.

## Compute Pressure API: Accessibility Considerations

As a starting point, the specification does not have an Accessibility Considerations section. Based on the a11y self-assessment the WG believes there are no known accessibility considerations related to the functionality exposed by the API.

# Accessibility Checklist

The [Accessibility Checklist](https://w3c.github.io/apa/fast/checklist.html) document is structured into the following sections, with top-level conditions reproduced here to facilitate WG review. Those with a checkmark are considered relevant to this specification and will be discussed in more detail in the sections that follow.

>- [ ] If technology allows visual rendering of content
>- [ ] If technology provides author control over color
>- [ ] If technology provides features to accept user input
>- [ ] If technology provides user interaction features
>- [ ] If technology defines document semantics
>- [ ] If technology provides time-based visual media
>- [ ] If technology provides audio
>- [ ] If technology allows time limits
>- [ ] If technology allows text content
>- [ ] If technology creates objects that don't have an inherent text representation
>- [ ] If technology provides content fallback mechanisms, whether text or other formats
>- [ ] If technology provides visual graphics
>- [ ] If technology provides internationalization support
>- [ ] If technology defines accessible alternative features
>- [ ] If technology provides content directly for end-users
>- [x] If technology defines an API
>- [ ] If technology defines a transmission protocol

## If technology defines an API

>- If the API can be used for structured content, it provides features to represent all aspects of the content including hidden accessibility features.

N/A

>- If the API relies on user agents to generate a user interface, the specification provides guidance about accessibility requirements needed to enable full interaction with the API.

This API does not generate any user interface.

An early version of the API had a provision for user agents to display a permission prompt when information from this API was requested. Based on implementer feedback the permission prompt gating for this API was removed.

This suggests no accessibility considerations to be documented in this specification.

**Comment by anssiko** [comment]
@kenchris please review together with relevant parties and suggest changes as appropriate. We will refer to this issue when we submit the a11y review request.

This is tracked as part of the wide review #177 for this API.

**Comment by anssiko** [comment]
Wide review has been completed as documented in #177. Thank you for your review and contributions.
